### PR TITLE
Gloves changes featuring nerfed shove cuffing

### PR DESCRIPTION
**Pull request by SkeletalElite**
## About The Pull Request
This PR increases cuff time to 4 seconds, but adds a trait for gloves that makes you cuff 1 second faster. Currently this is placed on black gloves, the captain's gloves and any other gloves which are used for combat (like krav maga, any tacklers, and combat gloves). Chameleon gloves also have this trait so that traitors have a glove choice other than the somewhat conspicuous combat gloves to help them with kidnapping, something they may do for objectives or contracting. 
It also makes nitrile and latex gloves hide your fingerprints once again.

## Why It's Good For The Game
Latex gloves not hiding fingerprints was basically done purely for balance reasons to make them not a direct upgrade over black gloves, however it bothered me that this made little sense logically and is very unintuitive and not something that's easy to tell in game. I've personally arrested multiple people while playing sec using finger print evidence they didn't even realize I could possibly have because they thought their gloves would protect them. 

Also I don't like playing against shove cuffing, dying because you dared to stand within 1 tile of a wall or table and got shoved once is unfun and takes basically zero effort or preparation and is one of the most effective things you can do to fight someone, it discourages RP because literally anyone can just turn on combat mode, walk you into the wall while you're typing and kill you with nothing but a pair a cable cuffs. This will make it take a little more prep and make you a little more conspicuous when you intend to do it.

## Changelog

:cl:
add: Wearing certain gloves (such as black gloves and combat oriented gloves) allow you to cuff people faster
balance: base handcuffing time is now 4 seconds
balance: latex gloves now hide your fingerprints
/:cl:

### CODE REVIEW

**Comment by starrm4nn** [comment]
does this include the cap, sec and antag modsuit gloves?

**Comment by SkeletalElite** [comment]
> does this include the cap, sec and antag modsuit gloves?

I forgot about them I'll add them too

**Comment by SkeletalElite** [comment]
Gave quick cuffing to a bunch of modsuits that it seemed like it made sense to be on. I'm not really too interested in touching modsuit balance a lot so I just made an unremovable module in sec/captain and antag/ert modsuits that gives the trait.

**Comment by Zergspower** [comment]
i got bamboozled by the latex gloves for thinking the exact same thing, never made sense lmao

**Comment by ExcessiveUseOfCobblestone** [comment]
generic black gloves having the cuff time buff is strange, I get why youre doing it but it's equally unintuitive that any ol crafted glove BUT ONLY IF ITS BLACK gives you this bonus just like the unwritten "well nitrile doesnt ACTUALLY cover your prints" mechanic. Almost makes sense to put it on the sec jumpsuit or make a "security glove" variant or something.

This also absolutely needs an in game hint such as on examine of clothes w this trait, you should not have to codedive to get a list of something that shaves off a fourth of the time to cuff someone.

**Comment by Admiral-Gigaton** [comment]
ok i like the idea of making it so you cant just shove cuff someone, but i feel the way you are doing it right now... isnt a very intelligent way. 

maybe just make it so the shover gets a little debuff that slows them a little? i mean, you just used a decent amount of energy shoving a full grown felinid into a wall, twice, and now you gotta force them into cuffs? youre gunna be a little tired or slowed with this, most likely. 

and the argument of "well buff people can do that and barely break a sweat" the average ss13 player isnt fucking baki hanma or that shit, you have to get augments, surgeries, all that shit just to do it.

and if you dont listen to the previous statement, just make sure to make gorilla, guerilla, tackle gloves, dolphin gloves, rocket gloves, maybe improvised tackle gloves, and CE gloves (its a combination of black gloves and insuls, so why wouldnt it be able to) able to cuff faster

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]
> This also absolutely needs an in game hint such as on examine of clothes w this trait, you should not have to codedive to get a list of something that shaves off a fourth of the time to cuff someone.

this part is required, the rest is debatable

**Comment by SomeoneYouProbablyKnow** [comment]
This feels like a half measure. You already want black gloves (seccies love toughing evidence so black gloves are invisible) and with this they also give combat benefits

Why not put the bonus on something more interesting like fingerless gloves so that the choice of glove is not automatic.
Sec can deal with the 1 extra second anyway.

**Comment by D4C-420** [comment]
More combat swapping with gloves when will it end. 

It would be better to increase the amount of time it takes for cable cuffs to be put on a target (encourages people to either steal proper cuffs from secure area's or go into maint and look from them in gutters, rather than grabbing a dime a dozen pair of "combat" gloves which is all over the station like free candy and leaving them on, changing nothing)

**Comment by SomeoneYouProbablyKnow** [comment]
> It would be better to increase the amount of time it takes for cable cuffs to be put on a target

Okay this is the superior idea

**Comment by SkeletalElite** [comment]
> More combat swapping with gloves when will it end.
> 
> It would be better to increase the amount of time it takes for cable cuffs to be put on a target (encourages people to either steal proper cuffs from secure area's or go into maint and look from them in gutters, rather than grabbing a dime a dozen pair of "combat" gloves which is all over the station like free candy and leaving them on, changing nothing)

Its not just about nerfing shove cuffing its about giving the standard black gloves a feature that differentiates them from latex and nitrile gloves. 

Additionally nerfing cable cuffs on top of this change isn't a bad idea, but is beyond the scope of this PR

**Comment by jughu** [comment]
making black gloves cuff faster is snowflake
and latex should reveal fingerprints but nittile should be a unique case (since its cmo's gloves)

**Comment by SomeoneYouProbablyKnow** [comment]
> standard black gloves a feature that differentiates them from latex and nitrile gloves.

Why is that a "good for the game" thing. Not everything has to have a mechanical component. They can just be black gloves. 

And they do have a special feature, security wears them which means they leave black glove fibers on everything.

**Comment by SkeletalElite** [comment]
> Why is that a "good for the game" thing. Not everything has to have a mechanical component. They can just be black gloves.

because the alternative is latex gloves leaving behind fingerprints which makes absolutely zero sense

**Comment by SomeoneYouProbablyKnow** [comment]
> because the alternative is latex gloves leaving behind fingerprints which makes absolutely zero sense

Sure. How does that transfer to having to make black gloves unique? and shive cuffing? If you want latex covering fingerprint back why not focus the change on that? Kinda looks like you made a whole PR about shove cuff just to buff latex gloves.

The suggestion about the type of cuffs infuencing cuff time is a better one. Black gloves making you better at zip tying would also make
> absolutely zero sense

**Comment by SkeletalElite** [comment]
> Sure. How does that transfer to having to make black gloves unique?

The entire reason latex gloves do not hide your fingerprints is because if they do, they are a direct upgrade over black gloves, particularly for security officers, since they allow you to fireman carry faster.  Gloves are meant to be sidegrades of each other. The only way to give latex gloves hiding fingerprints back is to give a feature to black gloves or to remove one from latex (faster fireman carry)

File: code/modules/clothing/gloves/special.dm [hunk 1]
```diff
@@ -102,7 +103,7 @@
 	greyscale_colors = null
 	siemens_coefficient = 0.3
 	armor_type = /datum/armor/latex_gloves
-	clothing_traits = list(TRAIT_QUICK_CARRY, TRAIT_FINGERPRINT_PASSTHROUGH)
```
Comment (Ghommie) [inline code/modules/clothing/gloves/special.dm:105]: Nitrile gloves don't have the `TRAIT_FINGERPRINT_PASSTHROUGH` trait in their `clothing_traits` variable by the by: https://github.com/tgstation/tgstation/blob/be9069ebc38732b8dc3061fc42c23ce437dc084c/code/modules/clothing/gloves/special.dm#L117

Comment (SkeletalElite) [reply code/modules/clothing/gloves/special.dm:105]: ah I was under the impression they were just a subtype of latex and were mostly the same thing. I'll update it

**Comment by kittysmooch** [comment]
> > Why is that a "good for the game" thing. Not everything has to have a mechanical component. They can just be black gloves.
> 
> because the alternative is latex gloves leaving behind fingerprints which makes absolutely zero sense

latex/nitrile gloves transfer prints in real life, nevermind that i'm too late to impact this PR. you have to layer them with cloth or other gloves to hide prints irl.

**Pull request merged**